flocker/node/functional/test_docker.py

```diff
@@ -205,6 +205,38 @@ def test_dead_is_listed(self):
                                  expected_states=(u'inactive',))
         return d
 
+    def test_list_with_missing_image(self):
+        """
+        ``DockerClient.list()`` can list containers whose image is missing.
+
+        The resulting output may be inaccurate, but that's OK: this only
```

Comment:
As described elsewhere, it's not actually limited to happening for non-running containers.